spec/jobs/update_topic_hot_scores_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "file_store/s3_store"
+
+RSpec.describe Jobs::UpdateTopicHotScores do
+  let :job do
+    subject
+  end
```

Comment:
```suggestion
  let(:job) { subject }
```

Reply:
this is automatic... not idea why the auto formatter did not catch this.